docs/installation.md

```diff
@@ -120,4 +130,3 @@
     ```
 
     If base-branch is not specified it will default to `main`.
```

Comment:
I think I would prefer that it gets its own section instead :) 

```suggestion
    If base-branch is not specified it will default to `main`.
    
=== "Aqua"

    ## Pre-requisites

    Install:

      - [Git](https://git-scm.com/downloads)
      - [Docker](https://docs.docker.com/get-docker/)
      - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
      - [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) OR [minikube](https://minikube.sigs.k8s.io/docs/start/)
      - [Argo CD CLI](https://argo-cd.readthedocs.io/en/stable/cli_installation/)

    ## Install binary with [aquaproj/aqua](https://github.com/aquaproj/aqua)
    `argocd-diff-preview` also supports installation by aqua. See [documentation](https://aquaproj.github.io/docs/install) for aqua setup instructions.

    ```bash
    aqua init
    aqua g -i dag-andersen/argocd-diff-preview
    aqua i
    argocd-diff-preview --help
    ```

    ## Usage

    You need to pull down the two branches you want to compare. The first branch will be cloned into the `base-branch` folder, and the other branch will be cloned into the `target-branch` folder.

    ```bash
    git clone https://github.com/<owner>/<repo> base-branch --depth 1 -q -b <branch-a>

    git clone https://github.com/<owner>/<repo> target-branch --depth 1 -q -b <branch-b>
    ```

    ## Run the binary
    ```bash

    argocd-diff-preview \
      --repo <owner>/<repo-name> \
      --base-branch <branch-a> \
      --target-branch <branch-b>
    ```

    If base-branch is not specified it will default to `main`.

```

Reply:
Thank you for your suggestion!